libqtile/layout/columns.py

```diff
@@ -103,23 +111,30 @@ class Columns(Layout):
         ("name", "columns", "Name of this layout."),
         ("border_focus", "#881111", "Border colour for the focused window."),
         ("border_normal", "#220000", "Border colour for un-focused windows."),
+        ("border_focus_stack", "#881111",
+         "Border colour for the focused window in stacked columns."),
+        ("border_normal_stack", "#220000",
+         "Border colour for un-focused windows in stacked columns."),
         ("border_width", 2, "Border width."),
         ("margin", 0, "Margin of the layout."),
-        ("autosplit", True, "Autosplit newly created columns."),
+        ("split", True, "New columns presentation mode."),
         ("num_columns", 2, "Preferred number of columns."),
         ("grow_amount", 10, "Amount by which to grow a window/column."),
         ("fair", False, "Add new windows to the column with least windows."),
+        ("insert_position", 1,
```

Comment:
What would you think using a boolean value, with `"insert_after", True` for example, since there are only two possible choices anyway?

Reply:
Also, if I understand correctly the tests diff, shouldn't we insert before by default, so that the current layout behavior is kept?

Reply:
`insert_position` is a little more general. The int value indicates the position relative to the current client. This argument comes from `base._ClientList.add(client, offset_to_current)`.

For example, it could be set to some large number to ensure that new clients are always inserted at the end of the list. It could also be set to a very small number (large negative) to ensure new clients are always at the beginning.

As for the default value, either way is fine. Inserting after is in line with `Wmii`'s default, which, in hindsight, made more sense to me.

Reply:
Thanks for the explanation. What would you think of using `1` on `Wmii`'s defaults, and `0` here?

Reply:
Yes, I believe that works. I'll go ahead and make the change.